app/services/getstream_webhook_service.rb

```diff
@@ -0,0 +1,97 @@
+# Parse getstream.io webhook reqest
+class GetstreamWebhookService
+  attr_reader :activity, :feed_id, :actor_id
+
+  def initialize(feed)
+    # TODO: See if multiple activities could actually happen
+    @activity = feed['new'].first
+    @feed_id = feed['feed'].split(':').last
+    @actor_id = @activity['actor'].split(':').last
+  end
+
+  # Find correspond user from feed id
+  def feed_target
+    User.find_by(id: feed_id)
+  end
+
+  # Get the corresponding feed url
+  def feed_url
+    model_type, model_id = activity['foreign_id'].split(':')
+    protocol = Rails.env.development? ? 'http://' : 'https://'
+    host = ActionMailer::Base.default_url_options.values.join(':')
+    base_url = "#{protocol}#{host}"
+    path = ''
+    case model_type
+    when 'Follow'
+      path = "/users/#{activity['actor'].split(':').last}"
+    when 'Post'
+      path = "/posts/#{model_id}"
+    when 'Comment'
+      path = "/comments/#{model_id}"
+    when 'PostLike'
+      target_id = activity['target'].split(':').last
+      path = "/posts/#{target_id}"
+    when 'CommentLike'
+      target_id = activity['target'].split(':').last
+      path = "/comments/#{target_id}"
+    end
+
+    "#{base_url}#{path}"
+  end
+
+  # Express activity of this feed in desired locale
+  def stringify_activity
+    locale = (feed_target&.language || 'en').to_sym
+    actor_name = User.find_by(id: actor_id)&.name
+    activity_str = {}
+
+    case activity['verb']
+    when 'follow'
+      activity_str[locale] = I18n.t(:followed,
+        scope: [:notifications],
+        actor: actor_name,
+        locale: locale)
+    when 'post'
+      activity_str[locale] = I18n.t(:post_mentioned,
+        scope: [:notifications],
+        actor: actor_name,
+        locale: locale)
+    when 'post_like'
+      activity_str[locale] = I18n.t(:post_like,
+        scope: [:notifications],
+        actor: actor_name,
+        locale: locale)
+    when 'comment_like'
+      activity_str[locale] = I18n.t(:comment_like,
+        scope: [:notifications],
+        actor: actor_name,
+        locale: locale)
+    when 'comment'
```

Comment:
Looks like the code below doesn't take into account posts that are followed by users. 

You'll be able to tell whether the notification is a mention or just a new comment to a post being followed by looking at the `mentioned_users` list that contains the ids of users mentioned in the comment.